web/app_test.php

```diff
@@ -1,12 +1,23 @@
 <?php
 
 use eLife\Journal\AppKernel;
+use Liuggio\Fastest\Process\EnvCommandCreator;
 use Symfony\Component\Debug\Debug;
 use Symfony\Component\HttpFoundation\Request;
 
 require_once __DIR__.'/../app/autoload.php';
 Debug::enable();
 
+if (!empty($_GET[EnvCommandCreator::ENV_TEST_CHANNEL])) {
+    setcookie(EnvCommandCreator::ENV_TEST_CHANNEL, $_GET[EnvCommandCreator::ENV_TEST_CHANNEL]);
+    die;
```

Comment:
This is a bit annoying. Couldn't get setting a cookie manually in `Selenium2Driver` to work.

Reply:
It could be due to host/folder/secure settings not being aligned, this should perfectly produce the correct settings.